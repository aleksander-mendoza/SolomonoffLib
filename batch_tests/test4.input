g = (.* 1{'a'+ 2} .* 3 ) 1 * 4 | (.* 2{'b'+ 2} .* 3 ) 1 * 5 | (.* 3{'c'+ 2} .* 3 ) 1 * 6
/submatch g 'ergacdgjxackdfgacgdr' 1
/submatch g 'bbb' 1
/submatch g '' 1
/submatch g '' 2
/submatch g '' 3
/submatch g 'aaabbcccc' 1
/submatch g 'aaabbcccc' 2
/submatch g 'aaabbcccc' 3
/submatch g 'atyjytjyaabjybjytjcjtccc' 1
/submatch g 'atyjaabjytjbcytjccytjc' 2
/submatch g 'aaajtybbjtccjtycc' 3