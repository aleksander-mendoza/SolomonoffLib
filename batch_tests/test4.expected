longerMatchesHigherWeights
subtractNondet
import
ostiaConservative
reweight
rpni
rpniEdsm
clearOutput
identity
dropEpsilon
rpniMdl
dict
stringFile
ostiaMaxDeepOverlap
activeLearningFromDataset
rpniMealy
inverse
parseATT
ostiaMaxCompatibleInputsAndOutputs
compress
ostiaMaxCompatible
ostiaMaxCompatibleInputs
subtract
ostiaWithDomain
ostiaMaxOverlap
importATT
randomDFA
compose
ostia
@uppercase
@add
@lowercase
@extractGroup
@reverse
